examples/evaluate_eval_assist_direct_several_criterias.py

```diff
@@ -0,0 +1,73 @@
+from typing import Any
```

Comment:
Currently support criteria from dataset using Operator.  So this example can be removed, but add different way to load the criteria.